### ISSUE TITLE
Add Batch Normalization to appropriate networks

### ISSUE THREAD

**Issue by kamuda1** [arfc/annsa#28]
Needs to be added to dense network for sure. May also add it to the CNN.

Batch normalization is:
- can be seen as layer-by-layer data prepossessing 
- has been shown to smooth out the convergence to a learned algorithm
- at each layer of nodes, batch normalization takes a running average of the data the network has seen and re-normalizes it (with itself) so that the activations are normally distributed.
- acts as a form of regularization (slightly decreasing the capacity of the network)

We will know that we can close this issue when:

- TensorFlow functions that apply batch normalization to the neural network have been 
- tests have been added to confirm the performance of this step
  - a test needs to check performance of batch normalization post-training. There are currently a few issues with how batch normalization is implemented, see [this](https://towardsdatascience.com/pitfalls-of-batch-norm-in-tensorflow-and-sanity-checks-for-training-networks-e86c207548c8) and  [this](tensorflow/tensorflow/issues/14809).
